dockerlib/dockerlib/manifest.go

```diff
@@ -12,13 +12,17 @@ import (
 )
 
 type ImageManifest struct {
-	Name          string     `json:"name"`
-	BaseDir       string     `json:"-"`
-	Maintainers   []string   `json:"maintainers"`
+	Name          string          `json:"name"`
+	BaseDir       string          `json:"-"`
+	Maintainers   []string        `json:"maintainers"`
+	MajorVersions []MajorVersions `json:"major-versions"`
```

Comment:
I'm not a fan of `major-versions` as the key here, but couldn't think of anything better. Suggestions very welcome!